pallets/transaction-payment/src/lib.rs

```diff
@@ -1048,3 +1048,13 @@ where
 		Self::compute_actual_fee(len, &info, &post_info, Zero::zero())
 	}
 }
+
+pub struct TestLengthToFeeU64;
```

Comment:
Why is this needed ?

Reply:
It's used in `pallets/dactr/src/extensions/check_batch_transactions_mock.rs`.

Basically we cannot impl `WeightToFee` trait for `u64` so we need to create a wrapper first. 
Instead of every mock creating the same wrapper, I just created and placed a default test one in the pallet `lib.rs` so that it can be used everywhere.